build_tools/github/build_doc.sh

```diff
@@ -17,13 +17,28 @@ set -e
 # If the inspection of the current commit fails for any reason, the default
 # behavior is to quick build the documentation.
 
+if [ -n "$GITHUB_ACTION" ]
+then
+    # Map the variables for the new documentation builder to the old one
+    CIRCLE_SHA1=$(git log --no-merges -1 --pretty=format:%H)
+    CIRCLE_JOB=$GITHUB_JOB
+
+    if [ "$GITHUB_EVENT_NAME" == "pull_request" ]
+    then
+        CIRCLE_BRANCH=$GITHUB_HEAD_REF
+        CI_PULL_REQUEST=true
+    else
+        CIRCLE_BRANCH=$GITHUB_REF_NAME
+    fi
+fi
+
 get_build_type() {
     if [ -z "$CIRCLE_SHA1" ]
     then
         echo SKIP: undefined CIRCLE_SHA1
         return
     fi
-    commit_msg=$(git log --format=%B -n 1 $CIRCLE_SHA1)
+    commit_msg=$(git log --no-merges -1 --oneline $CIRCLE_SHA1)
```

Comment:
Do we need this change? (If we do, I think we still need the `--format=%B` otherwise extra information will be returned)

Reply:
Not really, as I have obtained the `--no-merges` commit hash in the previous statements. Reverting.

Thanks!